File: 2nd Sem/Study/Code/C  code/angryProfessor.c
#include<stdio.h>
int main(){
    int a[1000],n,i,t,count;
    scanf("%d",&n);
    scanf("%d",&t);
    for(i=0;i<n;i++){
        scanf("%d",&a[i]);
    }
    count=0;
    for(i=0;i<n;i++){
        if(a[i]<=0){
            count=count+1;
        }
    }
    if(count==t){
        printf("No, the class is not cancelled");
    }
    else{
        printf("Yes,the class is cancelled");
    }
}

File: 2nd Semester/Study/Code/Python Code/Angryprofessor.py
a=[]
s=int(input("Strength of the class: "))
t=int(input("Threshold value: "))
for i in range(s):
   a.append(int(input()))
count=0   
for i in range(s):
   if a[i]<=0:
      count=count+1
if count==t:
   print("No the class isn't cancelled")
else:
   print("Yes the class is cancelled")   